apps/examples/testcase/ta_tc/systemio/itc/Kconfig

```diff
@@ -14,20 +14,79 @@ config SYSIO_ITC_UART
 	bool "Sysio UART TC"
 	default n
 
+if SYSIO_ITC_UART
+config SYSIO_ITC_UART_FILE_PATH
+	string "Sysio UART virtual test port"
+	default "/dev/ttyS1"
+endif
+
 config SYSIO_ITC_SPI
 	bool "Sysio SPI TC"
 	default n
 
+if SYSIO_ITC_SPI
+config SYSIO_ITC_SPI_BUS
+	int "Sysio SPI test bus"
+	default 0
+
+config SYSIO_ITC_SPI_BITS_PER_WORD
+	int "Sysio SPI test bits per word"
+	default 8
+
+config SYSIO_ITC_SPI_CHIP_SELECT
+	int "Sysio SPI test chip select"
+	default 0
+
+config SYSIO_ITC_SPI_FREQUENCY
+	int "Sysio SPI test frequency"
+	default 12000000
+
+config SYSIO_ITC_SPI_MODE
+	int "Sysio SPI test mode"
+	default 0
+endif
+
 config SYSIO_ITC_I2C
 	bool "Sysio I2C TC"
 	default n
 
+if SYSIO_ITC_I2C
+config SYSIO_ITC_I2C_BUS
+	int "Sysio I2C test bus"
+	default 0
+endif
+
 config SYSIO_ITC_PWM
 	bool "Sysio PWM TC"
 	default n
 
+if SYSIO_ITC_PWM
+config SYSIO_ITC_PWM_DEVICE
+	int "Sysio PWM test device"
+	default 0
+
+config SYSIO_ITC_PWM_CHANNEL
+	int "Sysio PWM test channel"
+	default 1
+
+config SYSIO_ITC_PWM_PERIOD
+	int "Sysio PWM test period"
+	default 1000
+endif
+
 config SYSIO_ITC_GPIO
 	bool "Sysio GPIO TC"
 	default n
+
+if SYSIO_ITC_GPIO
+config SYSIO_ITC_GPIO1_NUM
```

Comment:
postfix 'NUM' is unnecessary I think, or 'PORT' is better option. How do you think?

Reply:
@thapav Could you reply Taejun's comment?

Reply:
Yes @Taejun-Kwon  we could remove it. Let me update it

Reply:
Modified, thanks.